_docker/environments/awestruct-pull-request/docker-compose.yml

```diff
@@ -102,9 +102,23 @@ services:
       - RHD_TEST_PROFILE
 
   docker_chrome:
-   image: selenium/node-chrome
+   image: selenium/node-chrome-debug
```

Comment:
Generally I'd say its best practice to specify the version of the Docker image you want to use rather than relying on `latest`. The problem with `latest` is that it is a moving target. What is working one day can suddenly stop working as the `latest` image pointer has been updated to something that we have not tested.

Therefore I'd suggest picking whatever version is the one you want, and hardcoding the image to that. So `selenium/node-chrome-debug:2.53.0` rather than just `selenium/node-chrome-debug`